admin/src/pages/settings.js

```diff
@@ -61,6 +61,7 @@ function Settings() {
     titleField: null,
     colorField: null,
     defaultDuration: 30,
+    defaultStartTime: '12:00',    
```

Comment:
as of now, there's no way for users to edit this so let's remove it and uses moment instead